libwallet/assets/dcr/dex_wallet.go

```diff
@@ -504,21 +515,21 @@ func (dw *DEXWallet) MatchAnyScript(ctx context.Context, blockHash *chainhash.Ha
 // AccountUnlocked returns true if the account is unlocked.
 // Part of the Wallet interface.
 func (dw *DEXWallet) AccountUnlocked(_ context.Context, _ string) (bool, error) {
-	return !dw.Wallet.IsLocked(), nil
+	return !dw.w.Locked(), nil
 }
 
 // LockAccount locks the specified account.
 // Part of the Wallet interface.
 func (dw *DEXWallet) LockAccount(_ context.Context, _ string) error {
-	dw.Wallet.LockWallet()
+	dw.helper.LockWallet()
```

Comment:
If there's a reason for using the helper method rather than the wallet method, please leave a comment here.

Reply:
Uhmm, not really. The previous implementation required using `Wallet.LockWallet` (not wallet.Wallet) due to some checks in this method(and others) that we didn't want to duplicate everywhere.

But maybe we can do without them now that we are using `wallet.Wallet`.

The same applies to other methods in the `WalletHelper` interface